sysutils/jq/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        stedolan jq 1.5 jq-
+github.setup        stedolan jq 1.6 jq-
 revision            1
```

Comment:
When you increase the version, the `revision` should be set back to zero, which usually means you can just remove the line as it is the default. However, now that it has already been merged, there is no need to change it anymore.